tools/c7n_azure/c7n_azure/resources/key_vault.py

```diff
@@ -23,3 +30,53 @@ class resource_type(ArmResourceManager.resource_type):
         service = 'azure.mgmt.keyvault'
         client = 'KeyVaultManagementClient'
         enum_spec = ('vaults', 'list', None)
+
+
+@KeyVault.filter_registry.register('whitelist')
+class WhiteListFilter(ValueFilter):
+    schema = type_schema('whitelist', rinherit=ValueFilter.schema)
+    graph_client = None
+
+    def __init__(self, data, manager=None):
+        super(WhiteListFilter, self).__init__(data, manager)
+        self.op = 'difference'
+
+    def __call__(self, i):
+        if 'access_policies' not in i:
+            client = self.manager.get_client()
+            vault = client.vaults.get(i['resourceGroup'], i['name'])
+            # Retrieve access policies for the keyvaults
+            access_policies = []
+            for policy in vault.properties.access_policies:
+                access_policies.append({
+                    'tenant_id': policy.tenant_id,
```

Comment:
For consistency we should maintain the same casing that is on the resource already (should be camelCasing).